tests/ZiraLink.Api.Application.UnitTests/Services/CustomerServiceTest.cs

```diff
@@ -89,52 +91,310 @@ public async Task CreateLocallyAsync_WhenEverythingIsOk_ShouldBeSucceeded(string
+        {
+            var mockHttpTools = new Mock<IHttpTools>();
+
+
+            Assert.True(string.IsNullOrWhiteSpace(currentPassword));
+            CustomerService customerService = new CustomerService(_testTools.AppMemoryDbContext, mockHttpTools.Object);
+
+            var exception = await Assert.ThrowsAsync<ArgumentNullException>(() => customerService.ChangePasswordAsync(userId, currentPassword, newPassword, CancellationToken.None));
+            Assert.Equal("currentPassword", exception.ParamName);
+        }
+
+        [Theory]
+        [InlineData("3", "TestPassword", "")]
+        public async Task ChangePassword_WhenNewPasswordIsEmpty_ShouldBeFailed(string userId, string currentPassword, string newPassword)
+        {
+            var mockHttpTools = new Mock<IHttpTools>();
+
+
+            Assert.True(string.IsNullOrWhiteSpace(newPassword));
+            CustomerService customerService = new CustomerService(_testTools.AppMemoryDbContext, mockHttpTools.Object);
+
+            var exception = await Assert.ThrowsAsync<ArgumentNullException>(() => customerService.ChangePasswordAsync(userId, currentPassword, newPassword, CancellationToken.None));
+            Assert.Equal("newPassword", exception.ParamName);
+        }
+
+        [Theory]
+        [InlineData("9999", "TestPassword", "TestNewPassword")]
+        public async Task ChangePassword_WhenCustomerIdIsNotExist_ShouldBeFailed(string userId, string currentPassword, string newPassword)
+        {
+            var mockHttpTools = new Mock<IHttpTools>();
+
+
+            CustomerService customerService = new CustomerService(_testTools.AppMemoryDbContext, mockHttpTools.Object);
+
+            var exception = await Assert.ThrowsAsync<NotFoundException>(() => customerService.ChangePasswordAsync(userId, currentPassword, newPassword, CancellationToken.None));
+            Assert.Equal("Customer", exception.Message);
+        }
+
+        #endregion
+
+        #region UpdateProfile
+        [Theory]
+        [InlineData("3", "Test", "Testi")]
+        public async Task UpdateProfile_WhenEverythingIsOk_ShouldBeSucceeded(string userId, string name, string family)
+        {
+            var mockHttpClient = new Mock<HttpClient>(); 
+            var mockHttpTools = new Mock<IHttpTools>();
+       
+            var idsData = new ApiResponse<string> { Data = "10", Status = true };
+ 
+            mockHttpTools.Setup(p => p.InitializeHttpClientAsync(It.IsAny<CancellationToken>())).ReturnsAsync(mockHttpClient.Object);
+            mockHttpTools.Setup(p => p.CallIDSApis<ApiResponse<string>>(mockHttpClient.Object, It.IsAny<string>(), It.IsAny<object>(), It.IsAny<HttpMethod>(), It.IsAny<CancellationToken>())).ReturnsAsync(idsData);
+
+            var customerService = new CustomerService(_testTools.AppMemoryDbContext, mockHttpTools.Object);
+ 
+            await customerService.UpdateProfileAsync(userId, name, family, CancellationToken.None);
+
+            Assert.True(true); 
```

Comment:
Did you forget this?

Reply:
What's wrong? I couldn't catch you.

Reply:
`Assert.True(true);` makes no sense! it always gets passed and the input and algorithm never affect it, you should either remove it or replace it with a proper assert statement.